spec/functions/fqdn_rotate_spec.rb

```diff
@@ -4,9 +4,9 @@
 
 describe 'fqdn_rotate' do
```

Comment:
```suggestion
describe 'stdlib::fqdn_rotate' do
```

Reply:
This was a deliberate choice to show that the shim does actually correctly invoke the namespaced function.